src/test/java/uk/gov/companieshouse/exemptions/util/ExemptionsReadConverterTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.exemptions.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.bson.Document;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.exemptions.CompanyExemptions;
+import uk.gov.companieshouse.api.exemptions.CompanyExemptions.KindEnum;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
No mocks are used so don't need the MockitoExtension or `@BeforeEach` section. Instantiate class under test inline.